android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMHardwareKeyboardInterpreter.java

```diff
@@ -161,8 +167,15 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
       return false;
     }
 
+    int code;
+    if ((keyCode == KeyEvent.KEYCODE_BACKSLASH ) && (event.getScanCode() == SCAN_CODE_KEY_102ND) ) {
```

Comment:
Note: If the user sets their physical AZERTY keyboard to French, 
the 102nd key generates `KeyEvent.KEYCODE_PLUS` (scan code also = `SCAN_CODE_KEY_102ND`)